.github/workflows/conda-style-builds.yml

```
@@ -0,0 +1,84 @@
+name: Conda-compatible builds
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+defaults:
+  run:
+    # Enable Conda environment by using the login shell:
+    shell: bash -leo pipefail {0}
+
+jobs:
+  CI:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-24.04, macos-13, macos-latest, windows-2022]
+    env:
+      FC: gfortran
+      # FFLAGS: "-std=f2008"
+      CMAKE_BUILD_PARALLEL_LEVEL: 8
+      VERBOSE:
+      CTEST_PARALLEL_LEVEL: 8
+      CTEST_OUTPUT_ON_FAILURE: 1
+    runs-on: ${{ matrix.os }}
+    steps:
+      #
+      # Check out repository under $GITHUB_WORKSPACE
+      #
+      - name: Check out code
+        uses: actions/checkout@v4
+      #
+      # Cache Conda packages
+      #
+      - name: Cache Conda packages
+        uses: actions/cache@v4
+        with:
+          path: ~/conda_pkgs_dir
+          key: conda-pkgs-${{ matrix.os }}
+      #
+      # Set up Conda
+      #
+      - name: Set up Conda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          miniforge-version: latest
+          activate-environment: rte_rrtmgp_test
+          environment-file: environment-noplots.yml
+          python-version: 3.11
+          auto-activate-base: false
+          conda-remove-defaults: true
+          # Use the cache properly:
+          use-only-tar-bz2: false
+      #
+      # Install compiler and dependencies
+      #
+      - name: Install compiler, dependencies
+        run: |
+          conda install -c conda-forge ${FC} netcdf-fortran ninja -y
```

The style check probably wants this (no idea why it doesn't fail locally but it happened to me as well):
```suggestion
          conda install -c conda-forge "${FC}" netcdf-fortran ninja -y
```